Do not change the ABI when enabling or not traffic trace
--- a/module/owlib/src/c/ow_traffic.c
+++ b/module/owlib/src/c/ow_traffic.c
@@ -32,20 +32,27 @@
 	return NO_CONNECTION ;
 }
 
+#endif /* OW_SHOW_TRAFFIC */
+
 void TrafficOut( const char * data_type, const BYTE * data, size_t length, const struct connection_in * in )
 {
+#if OW_SHOW_TRAFFIC
 	fprintf(stderr, "TRAFFIC OUT <%s> bus=%d (%s)\n", SAFESTRING(data_type), in->index, DEVICENAME(in) ) ;
 	_Debug_Bytes( in->adapter_name, data, length ) ;
+#endif /* OW_SHOW_TRAFFIC */
 }
 
 void TrafficIn( const char * data_type, const BYTE * data, size_t length, const struct connection_in * in )
 {
+#if OW_SHOW_TRAFFIC
 	fprintf(stderr, "TRAFFIC IN  <%s> bus=%d (%s)\n", SAFESTRING(data_type), in->index, DEVICENAME(in) ) ;
 	_Debug_Bytes( in->adapter_name, data, length ) ;
+#endif /* OW_SHOW_TRAFFIC */
 }
 
 void TrafficOutFD( const char * data_type, const BYTE * data, size_t length, FILE_DESCRIPTOR_OR_ERROR file_descriptor )
 {
+#if OW_SHOW_TRAFFIC
 	struct connection_in * in = Bus_from_file_descriptor( file_descriptor ) ;
 	if ( in != NO_CONNECTION ) {
 		TrafficOut( data_type, data, length, in ) ;
@@ -53,10 +60,12 @@
 		fprintf(stderr, "TRAFFIC OUT <%s> file descriptor=%d\n", SAFESTRING(data_type), file_descriptor ) ;
 		_Debug_Bytes( "FD", data, length ) ;
 	}
+#endif /* OW_SHOW_TRAFFIC */
 }
 
 void TrafficInFD( const char * data_type, const BYTE * data, size_t length, FILE_DESCRIPTOR_OR_ERROR file_descriptor )
 {
+#if OW_SHOW_TRAFFIC
 	struct connection_in * in = Bus_from_file_descriptor( file_descriptor ) ;
 	if ( in != NO_CONNECTION ) {
 		TrafficIn( data_type, data, length, in ) ;
@@ -64,6 +73,6 @@
 		fprintf(stderr, "TRAFFIC IN  <%s> file descriptor=%d\n", SAFESTRING(data_type), file_descriptor ) ;
 		_Debug_Bytes( "FD", data, length ) ;
 	}
+#endif /* OW_SHOW_TRAFFIC */
 }
 
-#endif /* OW_SHOW_TRAFFIC */
--- a/module/owlib/src/include/ow_traffic.h
+++ b/module/owlib/src/include/ow_traffic.h
@@ -16,25 +16,13 @@
 /* Debugging interface to showing all bus traffic
  * You need to configure compile with
  * ./configure --enable-owtraffic
+ * if you do not want to get empty functions
  * */
 
-/* Show bus traffic in detail (must be configured into the build) */
-#if OW_SHOW_TRAFFIC
-
 void TrafficOut( const char * data_type, const BYTE * data, size_t length, const struct connection_in * in );
 void TrafficIn( const char * data_type, const BYTE * data, size_t length, const struct connection_in * in );
 
 void TrafficOutFD( const char * data_type, const BYTE * data, size_t length, FILE_DESCRIPTOR_OR_ERROR file_descriptor );
 void TrafficInFD( const char * data_type, const BYTE * data, size_t length, FILE_DESCRIPTOR_OR_ERROR file_descriptor );
 
-#else /* OW_SHOW_TRAFFIC */
-
-#define TrafficOut( data_type, data, length, in ) { } while (0)
-#define TrafficIn( data_type, data, length,  in ) { } while (0)
-
-#define TrafficOutFD( data_type, data, length, file_descriptor ) { } while (0)
-#define TrafficInFD( data_type, data, length,  file_descriptor ) { } while (0)
-
-#endif /* OW_SHOW_TRAFFIC */
-
 #endif							/* OW_TRAFFIC_H */
